ui-core/src/components/inputs/TolerancePicker.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState, useRef, useMemo } from 'react';
+import Input, { InputProps } from './Input';
+import Modal from '../Modal';
+
+export type ToleranceValues = {
+  minusTolerance: number;
+  plusTolerance: number;
+};
+
+type ToleranceRange = { label: string; value: number };
+
+export type TolerancePickerProps = InputProps & {
```

Comment:
Do you think we can Omit 'value' in InputProps ? Since we handle the value with toleranceValues
```suggestion
export type TolerancePickerProps = Omit<InputProps, 'value'> & {
```